backend/README.MD

```diff
@@ -0,0 +1,77 @@
+# MSSE663S8W1GroupProject Backend
+Download and install Postman for easy testing.
+Make sure you setup MongoDB with a database named "*recipe-book*"
+
+## Commands to Run Development
```

Comment:
The postman steps are now completely optional. This will just show you how angular will communicate with the backend and how the backend will respond.

Reply:
If I'm not mistaken, MongoDB should create the database if it doesn't exist upon first connection. Have you done something otherwise to prevent this?